webapp/components/PageFooter/PageFooter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="footer" class="ds-footer">
+    <a href="https://human-connection.org" target="_blank" v-html="$t('site.made')"></a>
+    <span>-</span>
+    <nuxt-link to="/imprint">{{ $t('site.imprint') }}</nuxt-link>
+    <span>-</span>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-06T07:06:23Z" title="Friday, September 6th 2019, 9:06:23 am +02:00">Sep 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,15 @@
+<template>
+  <div id="footer" class="ds-footer">
+    <a href="https://human-connection.org" target="_blank" v-html="$t('site.made')"></a>
+    &nbsp;-&nbsp;
+    <nuxt-link to="/imprint">{{ $t('site.imprint') }}</nuxt-link>
+    &nbsp;‑&nbsp;
```


… this `‑` (dash) and see the others below … 👇🏼